src/__internal__/field-help/field-help.spec.tsx

```diff
@@ -1,9 +1,9 @@
 import React from "react";
 import { mount } from "enzyme";
-import FieldHelp from "./field-help.component";
+import FieldHelp, { FieldHelpProps } from "./field-help.component";
 import { assertStyleMatch } from "../../__spec_helper__/test-utils";
 
-function render(props) {
+function render(props?: FieldHelpProps) {
```

Comment:
I appreciate this was not part of this ticket, but I noticed on line 29 we have a `inputWidth` prop which doesn't exist in `FieldHelpProps`. Looking at the prop definitions I think `inputWidth` needs to be `labelWidth`. Would you be able to add this? This may allow you to workaround the dynamic prop definition in `FieldHelpProps` if this was the reason behind it :)

Reply:
I was a little confused by it as it's asserting the default value for the width and if I update it the asserted styles will be incorrect. I suspect it's a test that was left after some refactor work but I'll leave it in as it asserts the default and confirms the rest props are working as expected

Reply:
I've removed the rest interface so have updated the test, one for default width the other for a custom value